bindings/apple/Tests/MatrixRustSDKTests/ClientTests.swift

```diff
@@ -23,24 +23,14 @@ final class ClientTests: XCTestCase {
         }
     }
     
-    func testBuildingWithUsername() {
```

Comment:
This one was pinging matrix.org so removed it.

Reply:
Ouch, good catch.